src/System.Management.Automation/engine/PSConfiguration.cs

```diff
@@ -33,30 +33,47 @@ public enum ConfigScope
     /// Reads from and writes to the JSON configuration files.
     /// The config values were originally stored in the Windows registry.
     /// </summary>
+    /// <remarks>
+    /// The config file access APIs are designed to avoid hitting the disk as much as possible.
```

Comment:
In previous PRs we said that the config file contains security sensitive options and we should read them immediately. Now we ignore this. Proposal is to reset the cache by timer per some minutes.

Reply:
> Now we ignore this. 

I don't understand this. The first time we read a config file, all settings are cached for later use, including the security sensitive settings, so it is read immediately.

> Proposal is to reset the cache by timer per some minutes.

Why? We should never attempt to re-read policy settings as they may be changed in between and contains contradict settings.

Reply:
I am thinking about GPO behavior. If GPO is changed new values are applied intermediately at runtime. I'd expect the same behavior for config file too.